inc/config.php

```diff
@@ -47,33 +47,19 @@
 
 Config::detectRootDoc();
 
+loadGlpiConfig((isset($USEDBREPLICATE) ? $USEDBREPLICATE : 0),
+      (isset($DBCONNECTION_REQUIRED) ? $DBCONNECTION_REQUIRED : 0),
+      (isset($TRY_OLD_CONFIG_FIRST) ? 1 : 0));
 
-if (!file_exists(GLPI_CONFIG_DIR . "/config_db.php")) {
-   Session::loadLanguage();
-   // no translation
-   if (!isCommandLine()) {
-      Html::nullHeader("DB Error", $CFG_GLPI["root_doc"]);
-      echo "<div class='center'>";
-      echo "<p>Error: GLPI seems to not be configured properly.</p>";
-      echo "<p>config_db.php file is missing.</p>";
-      echo "<p>Please restart the install process.</p>";
-      echo "<p><a class='red' href='".$CFG_GLPI['root_doc']."'>Click here to proceed</a></p>";
-      echo "</div>";
-      Html::nullFooter();
+function loadGlpiConfig($USEDBREPLICATE, $DBCONNECTION_REQUIRED, $TRY_OLD_CONFIG_FIRST) {
```

Comment:
i agree, I left this way because the change is big enough to make the diff rather hard to read. If tis proposal may be acceptable, then I think it is time to rename them.